docs/index.md

```diff
@@ -1,2 +1,15 @@
 Here you will find a series of docs that outline various procedures and how-tos
```

Comment:
Can you please mention the [`PV` setup docs](https://github.com/operate-first/continuous-deployment/blob/master/docs/downstream/on-cluster-persistent-storage/README.md) in here as well?